threadbare/operations.py

```diff
@@ -527,6 +528,48 @@ def prompt(msg):
         return input("> ")
 
 
+#
+# uploads and downloads
+#
+
+
+def _transfer_fn(client, direction, **kwargs):
+    base_kwargs = {
+        # TODO: support an 'overwrite?' option?
```

Comment:
scp and sftp apparently differ in their default behaviour regarding overwrites. Confirm what threadbare is doing (should be dying instead of overwriting).

Reply:
I think this needs to be made explicit and handled in a separate issue

Reply:
https://github.com/elifesciences/threadbare/issues/35